hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```
@@ -8975,6 +8975,40 @@ objects:
           resendWait: 24
           resolvedBody: Your cluster's cluster-wide proxy is no longer raising any
             errors concerning network egress. No further action on this issue is required.
+    - apiVersion: ocmagent.managed.openshift.io/v1alpha1
+      kind: ManagedNotification
+      metadata:
+        name: sre-upgrade-managed-notifications
+        namespace: openshift-ocm-agent-operator
+      spec:
+        notifications:
+        - name: UpgradeControlPlaneUpgradeTimeout
+          severity: Error
+          summary: Control plane upgrade completion delayed
+          activeBody: There has been a delay observed in the anticipated completion
+            time of your cluster's Control Plane upgrade. Red Hat SRE has been notified
+            and will commence the investigation.
+          resendWait: 12
```

IMO this is not worth sending every 12 hours. I'd almost say we either don't want to resend it at all (as we're not asking the cluster owner to change anything), or at least every 72 hours. I'm thinking of situations where an upgrade is stuck due to circumstances beyond an SRE's control to fix. (ie. owner-induced issues). Since "Never resend" is not something we can do in OCM Agent right now, a longer window would have to be the alternative.

There is a flaw in this plan in that if a cluster upgrades -> times out -> gets unblocked -> upgrades again within 72 hours and times out again -> the SL is never sent until after the 72 hour period. But we don't have to be perfect here.

Given the above, we should slightly rephrase the text so that it doesn't sound like we're 'commencing' an investigation each time it's sent.

```suggestion
          activeBody: There has been a delay observed in the anticipated completion
            time of your cluster's Control Plane upgrade. Red Hat SRE has been notified
            of the situation.
          resendWait: 72
```